sdk/cosmos/azure-cosmos/README.md

```diff
@@ -1,133 +1,149 @@
 ﻿# Azure CosmosDB Client Library for Java
 
-[![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-cosmos.svg)](https://search.maven.org/artifact/com.microsoft.azure/azure-cosmos)
```

Comment:
link to the sdk on maven is good to have. any reason it is moved? can we add it back?

Reply:
I will add it back, no specific reason to remove it.

Reply:
Done.